homeassistant/components/hue/light.py

```diff
@@ -182,12 +195,20 @@ def async_update_items(bridge, api, current, async_add_entities, create_item):
 class HueLight(Light):
     """Representation of a Hue light."""
 
-    def __init__(self, coordinator, bridge, is_group, light):
+    def __init__(
+        self,
+        coordinator,
+        bridge,
+        is_group,
+        light,
+        supported_features=SUPPORT_HUE_EXTENDED,
```

Comment:
We should not have a default here. Let that be decided by the code that creates the class.